js/battledata.js

```diff
@@ -847,8 +847,13 @@ var Tools = {
 			facing = 'back';
 		}
 
+		// Decide what gen sprites to use.
+		var gen = {1:'rby', 2:'gsc', 3:'rse', 4:'dpp', 5:'bw', 6:'xy'}[options.gen];
+		if (Tools.prefs('nopastgens')) gen = 'xy';
+		if (Tools.prefs('bwgfx') && gen === 'xy') gen = 'bw';
```

Comment:
Why only for gen 6? o_O

Why do we only have an override for gen 6 -> 5 anyway?